freeipa.spec.in

```diff
@@ -1079,6 +1080,13 @@ rm -f %{buildroot}%{_usr}/share/ipa/ui/images/product-name.png
 %endif
 # RHEL spec file only: END
 
+# Register CLI tools for bash completion
+for clitool in ipa-migrate
+do
+    register-python-argcomplete "${clitool}" > "${clitool}"
```

Comment:
This actually needs a `BuildRequires: python3-argcomplete` because otherwise the tool is not available during the package build and the whole build fails:

```
+ for clitool in ipa-migrate
+ register-python-argcomplete ipa-migrate
/var/tmp/rpm-tmp.wfpGYR: line 81: register-python-argcomplete: command not found
```

Reply:
> This actually needs a `BuildRequires: python3-argcomplete` because otherwise the tool is not available during the package build and the whole build fails:
> 
> ```
> + for clitool in ipa-migrate
> + register-python-argcomplete ipa-migrate
> /var/tmp/rpm-tmp.wfpGYR: line 81: register-python-argcomplete: command not found
> ```

Fixed